src/hearings/converters/panel-roles.answer.converter.ts

```diff
@@ -0,0 +1,40 @@
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { LovRefDataModel } from '../models/lovRefData.model';
+import { State } from '../store';
+import { AnswerConverter } from './answer.converter';
+
+export class PanelRolesAnswerConverter implements AnswerConverter {
+  constructor(protected readonly route: ActivatedRoute) { }
+
+  public transformAnswer(hearingState$: Observable<State>): Observable<string> {
+    const panelRoles: LovRefDataModel[] = this.route.snapshot.data.otherPanelRoles;
+
+    return hearingState$.pipe(
+      map(state => {
+        const panelRequirements = state.hearingRequest.hearingRequestMainModel.hearingDetails.panelRequirements;
+        const panelSpecialism: string[] = panelRequirements && panelRequirements.panelSpecialisms;
+        const selectedSpecialisms: string[] = [];
+        panelSpecialism.forEach((specialismName) => {
+          let selectedSpecialismName: string = '';
+          panelRoles.forEach(role => {
+            if (role.key === specialismName) {
+              selectedSpecialismName = role.value_en;
+              return;
```

Comment:
Avoid using return in forEach loop, as using return in a forEach() is equivalent to a [continue](https://masteringjs.io/tutorials/fundamentals/foreach-continue) in a conventional loop. Use (for of) instead if breaking is required.